package.json

```diff
@@ -1,5 +1,8 @@
 {
   "devDependencies": {
     "big-integer": "^1.6.26"
+  },
+  "dependencies": {
+    "decimal.js": "^9.0.1"
```

Comment:
Sort of related to @natefaubion's comment in https://github.com/awakesecurity/purescript-precise-datetime/pull/7#pullrequestreview-100276164 about making this internal, I think this should be in `devDependencies`.

Normally we'd only need this to run the tests (so `devDependencies` makes sense for that) but otherwise I think it better communicates that we're expecting people to provide this themselves somehow.

Reply:
Moved to `devDependencies`

(Not sure why npm added it to dependencies even though i specified `--save-dev`...)